modules/nodes-nodes-garbage-collection-configuring.adoc

```diff
@@ -16,11 +16,11 @@ As an administrator, you can configure how {product-title} performs garbage coll
 
 You can configure any combination of the following:
 
-* soft eviction for containers
-* hard eviction for containers
-* eviction for images
+* Soft eviction for containers.
```

Comment:
I don't think the period is necessary for these 2 list items since it isn't a sentence